httpx/_client.py

```diff
@@ -626,7 +627,12 @@ def __init__(
             )
             for key, proxy in proxy_map.items()
         }
-        self._proxies = dict(sorted(self._proxies.items()))
+        if mounts is not None:
```

Comment:
Do I understand correctly that the mounts overrides proxies? Like:
```python
client = Client(..., proxies={"http://": SomeTransport()}, ..., mounts={"http://": SomeOtherTransport())
assert isinstance(client._mounts["http://"], SomeOtherTransport)
```
Do we need to outline it somewhere?